app/assets/stylesheets/application.scss

```diff
@@ -264,6 +264,11 @@ ul.with-bullets {
   color: $color-red;
 }
 
+.text--grey-bold {
```

Comment:
is there an existing class for this somewhere? i did a brief search but didn't find one.

Reply:
it's my preference to not use red & grey in these labels but I'm not feeling creative right now so idk what good names would be

Reply:
i have very strong opinions about this, but it's not the right time to open that particular can of worms.